relay-server/src/actors/envelopes.rs

```diff
@@ -1756,7 +1787,10 @@ impl EnvelopeProcessor {
         self.process_sessions(state);
         self.process_client_reports(state);
         self.process_user_reports(state);
-        self.process_profiling_items(state);
+
+        if_processing!({
+            self.process_profiling_items(state);
```

Comment:
This now means that you do not drop profiles in downstream Relays (e.g. PoPs) if the feature flag is disabled. Is that intentional?

My previous comment may have been misleading, I was referring to just the parsing / validation part. Your choice; I have a slight preference to keep dropping the items early when the feature is disabled, even without fully parsing them.

Reply:
So now, it will drop them as soon as possible if the feature flag is not enabled and will only parse if the processing feature is enabled. Is that what you suggested?

Reply:
Yes, that sounds reasonable.